Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/container/ContainerRegistryMvpModel.java

```diff
@@ -101,16 +102,16 @@ public PrivateRegistryImageSetting createImageSettingWithRegistry(@NotNull final
         if (!registry.adminUserEnabled()) {
             throw new Exception(ADMIN_USER_NOT_ENABLED);
         }
-        final RegistryListCredentials credentials = registry.listCredentials();
+        final RegistryCredentials credentials = registry.getCredentials();
         if (credentials == null) {
             throw new Exception(CANNOT_GET_CREDENTIAL);
         }
         String username = credentials.username();
-        final List<RegistryPassword> passwords = credentials.passwords();
+        final Map<AccessKeyType, String> passwords = credentials.accessKeys();
         if (Utils.isEmptyString(username) || passwords == null || passwords.size() == 0) {
```

Comment:
Now the type of passwords is changed from List to Map, so `passwords.size() != 0` cannot ensure `passwords.get(AccessKeyType.PRIMARY)` has a valid value in L144. I'm not sure whether it is possible that `passwords` contains only secondary key.